app/models/coin.rb

```diff
@@ -154,7 +154,11 @@ def live_market_data
   def market_info market_data = nil
     data = market_data || live_market_data.dup
     data["24h_volume_usd"] = humanize(data["24h_volume_usd"], '$') if data["24h_volume_usd"]
-    data["available_supply"] = humanize(data["available_supply"]) if data["available_supply"]
+    if self.available_supply then
```

Comment:
Whoa, didn't realize Ruby had a `then` keyword... very cool!

I don't quite see the value of its usage here though, shall we remove?

I think the only usage I think makes sense is a single line `if/else` statement, i.e.

```
if (1==2) then puts "Math doesn't work" else puts "Math works!" end
```

Credit to:
https://stackoverflow.com/questions/5659360/why-should-you-avoid-the-then-keyword-in-ruby

Reply:
Yeah, seems to be a way to fix ambiguous if statements that's not needed here. Thanks.